### PR TITLE
Update for Scala 2.13

### PR DESCRIPTION
**Pull request by dcsobral**

### CODE REVIEW

File: _overviews/macros/annotations.md [hunk 1]
```diff
@@ -16,11 +16,12 @@ permalink: /overviews/macros/:title.html
 
 **Eugene Burmako**
 
-Macro annotations are only available with the macro paradise plugin (in Scala 2.10.x, 2.11.x and 2.12.x alike).
-Their inclusion in official Scala might happen in Scala 2.13, but there is no certainty about it yet.
-Follow the instructions at the ["Macro Paradise"](paradise.html) page to download and use our compiler plugin.
+Macro annotations are available in Scala 2.13 with the `-Ymacro-annotations` flag, but only with the macro paradise
```
Comment (Philippus) [inline _overviews/macros/annotations.md]: The second part is a hard to parse for me. Suggestion: `, and with the macro paradise plugin from Scala 2.10.x to Scala 2.12.x`

**Comment by SethTisue** [comment]
@dcsobral interested in updating this based on the review feedback?

**Comment by dcsobral** [comment]
Uh, you guys can alter the PR directly. I think. :) Since I edited the file
directly on github to create the PR, I don't have these changes locally.
-- 
Daniel C. Sobral

**Comment by Philippus** [comment]
I'll merge this one and my suggestion in another PR. Thank you @dcsobral !
